docs/tutorial-1.md

```diff
@@ -0,0 +1,50 @@
+# How to create an Action
+
+## A first visit
+
+After all steps from the [installation](./installation.md) are running a visit to [http://localhost:8080](http://localhost:8080) should show the following UI:
+
+
+![Cicero WebUI Intro](./cicero_webui_intro.png "Cicero WebUI Intro")
+
+The first visit should also change the url to: [http://localhost:8080/action/current?active](http://localhost:8080/action/current?active)
+
+## Click on "Create New Action"
+After clicking the "Create New Action" button the url should change to:
+[http://localhost:8080/action/new](http://localhost:8080/action/new)
+
+And the UI should correspond to:
+![Cicero WebUI New Action](./cicero_webui_new_action.png "Cicero WebUI New Action")
+
+## Enter the path to the Actions, which should be created
+
+**For any url the 'https://' part needs to be omitted.**
+
+The path to the Actions can be a remote path like [github.com/input-output-hk/cicero](github.com/input-output-hk/cicero)
+or a local path like [/home/developer/IOHK/cicero](/home/developer/IOHK/cicero).
+
+**This doesn't need to be a path to Cicero itself:**
+
+Cicero Actions can also be part of a repository, which isn't Cicero.
+See [How to include external actions](./how-to-include-external-actions.md) for more information.
+
+### Assume the following path was entered:
```

Comment:
I think it would be helpful to clarify that we mean path where cicero is cloned or cicero URL github.com/input-output-hk/cicero

Reply:
Added an additional comment for this.